README.md

```diff
@@ -185,7 +189,7 @@ A current release is on the leading edge of our SDK development, and is intended
 
 | Version | Supported Environments  | State     | First Release | EOL/Terminated |
 |---------|-------------------------|-----------|---------------|----------------|
-| 3       | Node.js >= 16           | Supported | 01 Feb 2022   | TBD            |
+| 3       | Node.js >= 18           | Supported | 01 Feb 2022   | TBD            |
```

Comment:
I believe that with dropping support for node we should release a new major version instead of changing the version in the current one. So after all changes in this feature brach are done, we (oclif, node, TS) we should release a new major.